docs/guide/help.md

```diff
@@ -0,0 +1,45 @@
+completing its content. The website will replace the (rather incomplete) [User manual](https://github.com/xournalpp/xournalpp/wiki/User-Manual) on the Github repository.
+More technical questions (like how to build Xournal++ from source, or how to check out a pull request) that are relvant for technically more advanced users will still be
+presented in the [Github wiki](https://github.com/xournalpp/xournalpp/wiki).
+
+## Github discussions
+
+Our place for discussing all kind of matters related to Xournal++ is [Github discussions](https://github.com/xournalpp/xournalpp/discussions).
+This should be helpful for:
+
+- Asking questions about features
+- Sharing tips and tricks with other members of the community
+- Finding out what other people are using Xournal++ for
+- Getting help with installation or setting up a feature
+- etc.
+
+We hope to grow our community knowledge base here.
+
+## Github issue tracker
+
+Use the [issue tracker](https://github.com/xournalpp/xournalpp/issues) on the Github repository to report bugs and propose enhancements/new features.
+Please check beforehand if the same bug has already been reported or the same enhancement has already been suggested. If so, use the existing ticket to express yourself.
+This will make it easier for us to keep everything under control. Duplicates will usually be closed rapidly, but it always takes time to find them, which the develeopers
+would prefer to invest in coding and improving the documentation. Issues that are resolved will be closed as well. Open questions are better discussed on Github discussions.
+
+## Gitter channel
+
+If you want to discuss some topic related to Xournal++ in chat mode, you may find someone to talk with in the [Gitter channel](https://gitter.im/xournalpp/xournalpp).
+
+## The Crowdin project
+
+Our translations are managed in the [Crowdin project](https://crowdin.com/project/xournalpp). If you would like to contribute, create an account and start contributing.
+While some languages like English, German and Italian are basically complete, other languages require more care. If you are interested in translating a new language,
+contact us on Gitter or create a new issue and we will unlock the language on Crowdin.
+
+## Github pull requests
+
+If you want to contribute to the codebase of Xournal++ you may create a [pull request on Github](https://github.com/xournalpp/xournalpp/pulls). Once the pull request is
+ready for review, one of the developer team will review it, suggest changes if necessary or accept/decline the pull request. If you plan some feature, that has not been
+requested in the issue tracker yet, better open an issue there first, so you know whether your work will be appreciated. Pull requests can be linked to specific issues,
+that will get automatically closed when the pull request is merged.
```

Comment:
Last sentence here isn't really informative for contributors, so I would just remove it.